src/java.base/share/classes/java/io/Reader.java

```diff
@@ -140,6 +140,124 @@ public void close() {
         };
     }
 
+    /**
+     * Returns a new {@code Reader} whose source is a {@link CharSequence}.
```

Comment:
Probably better to more this more precise, e.g.  Returns a {@code Reader} that reads characters from a {@code CharSequence}, starting at the first character in the sequence.